bcda/client/bluebutton_test.go

```diff
@@ -452,8 +452,8 @@ func (s *BBRequestTestSuite) TestValidateRequest() {
 					"%s does not end with %s", req.Header.Get("BlueButton-OriginalUrl"), req.URL.String())
 				assert.Equal(t, req.URL.RawQuery, req.Header.Get("BlueButton-OriginalQuery"))
 
-				assert.Equal(t, jobID, req.Header.Get("BCDA-JOBID"))
-				assert.Equal(t, cmsID, req.Header.Get("BCDA-CMSID"))
+				assert.Equal(t, jobID, req.Header.Get("BULK-JOBID"))
```

Comment:
Should we add a negative test to verify we're no longer supplying the `BCDA-JOBID` and `BCDA-CMSID`?

Reply:
Would it matter if we still were?

Reply:
It shouldn't affect the success/failure of our requests, but would indicate that something in our code is unexpectedly still setting some old headers